CorsixTH/Lua/dialogs/fullscreen/town_map.lua

```diff
@@ -188,10 +188,8 @@ function UITownMap:draw(canvas, x, y)
   self.info_font:draw(canvas, radiators,    x +  95, y + 265)
 
   -- Heating costs
-  local heating_costs = math.floor(((hospital.heating.radiator_heat *10)* radiators)* 7.5)
-  if app.config.free_build_mode then  -- Everything is 0$ in free build mode
-    heating_costs = 0
-  end
+  local heating_costs = app.config.free_build_mode and 0  -- Everything is 0$ in free build mode
+			or math.floor(((hospital.heating.radiator_heat *10)* radiators)* 7.5)
```

Comment:
Second line uses TABs rather than spaces.  
Perhaps you should spend some time teaching your editor not to use them, to avoid having to fix it afterwards every time. Alternatively you can look for a more sane editor.  
Making your life more easy is always a good idea :)

Continuation lines (line 192 here) use a double indent (ie 4 spaces) from the left. That is, it should look like
```
local ...
    or ...
```

I am not sure about the comment in the middle of the `and/or` sequence. A reader may think the line is finished at the end of line 191.  Would it make sense to you to merge that information into the comment at line 190?   It's fine to rephrase the text.

The old code didn't have proper spacing around the multiplication operators. You can fix that as well while changing the code (there should be a space at both sides of a `*`).  
The old code has a few too many parentheses, eliminating them can also be done.  
These improvements are optional, but leaving the place a little cleaner than you found it is generally a good idea.

A more complicated change is that `math.floor(((hospital.heating.radiator_heat *10)* radiators)* 7.5)` here is used to display  (and only display) the heating costs. That means that elsewhere in the program the same formula must exist as well to actually compute the costs and deduct it from the hospital balance. Obviously, both spots should have the exact same formula or the displayed cost here is lying to the user.  
In programming, a general objective is to have everything exactly once. In other words, this formula should exist at one spot only, and be used both for displaying and for updating the hospital balance.
In that way the displayed amount is always the same as the deducted amount. That becomes very useful if we ever need to change the formula.  
When you feel up for it, you could try fixing that. Perhaps in a next patch? (it's not quite a "free build improvement in town map", the topic of this patch)

Always feel free to start a discussion in the chat. It's easy to get overwhelmed by all the things, a little guidance what to do can be very useful. We all started that way, and all advice is free :)